platzky/blog/blog.py

```
@@ -49,40 +59,44 @@ def post_comment(post_slug):
         )
         return get_post(post_slug=post_slug)
 
-    @blog.route("/<post_slug>", methods=["GET"])
-    def get_post(post_slug):
+    def _get_content_or_404(getter_func, slug, *exception_types):
```

Ruff (0.14.10)

Missing return type annotation for private function `_get_content_or_404`

(ANN202)